packages/legacy-client/src/client/objectSets/OsdkObjectSet.test.ts

```diff
@@ -75,6 +76,133 @@ describe("OsdkObjectSet", () => {
     });
   });
 
+  it("creates the count aggregation", async () => {
+    const os = createBaseTodoObjectSet(context);
+    mockAggregateResponse({ data: [] });
+    await os.count().compute();
+    expect(fetch).toHaveBeenCalledOnce();
+    expect(fetch).toHaveBeenCalledWith(
+      ...expectedJestResponse("Ontology/objectSets/aggregate", {
+        objectSet: { type: "base", objectType: "Todo" },
+        groupBy: [],
+        aggregation: [{ type: "count", name: "count" }],
+      }),
```

Comment:
These tests are only testing that the request payload is correct, not that it handles the response. I can dig up the response payloads and update them later.